build/images/Dockerfile.build.ubuntu

```diff
@@ -25,16 +25,21 @@ RUN --mount=type=cache,target=/go/pkg/mod/ \
 
 COPY . /antrea
 
-RUN --mount=type=cache,target=/go/pkg/mod/ \
-    --mount=type=cache,target=/root/.cache/go-build/ \
-    make antrea-agent antrea-controller antrea-cni
 # Disable CGO for antctl in case it is copied outside of the container image. It
 # also reduces the size of the binary and aligns with how we distribute antctl
 # in release assets.
 RUN --mount=type=cache,target=/go/pkg/mod/ \
     --mount=type=cache,target=/root/.cache/go-build/ \
     CGO_ENABLED=0 make antctl-linux && mv bin/antctl-linux bin/antctl
 
+RUN --mount=type=cache,target=/go/pkg/mod/ \
```

Comment:
nit: a comment would be nice to explain that we build `antrea-agent` and `antrea-cni` first in order to share an extra layer with `build/images/Dockerfile.build.agent.ubuntu`

Reply:
done, also made same changes to coverage and ubi images.